pygmalionanalysis.py

```diff
@@ -0,0 +1,75 @@
+# Load data from a file (will be part of your data processing script)
+import pickle
+import math
+import io
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+#tested setup by printing requests
+def setup():
+    import requests
+    shaw_full_text = requests.get('http://www.gutenberg.org/cache/epub/3825/pg3825.txt').text
```

Comment:
Data-fetching script (which is implemented here) and the data analysis script should be in different files so you don't make HTTP request everytime you run an analysis.